resources/script/ace-builds/demo/bookmarklet/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
do you really need to check in all the project? remember this will download to the phone potentially